rs/tests/consensus/tecdsa/utils/BUILD.bazel

```
@@ -0,0 +1,43 @@
+load("@rules_rust//rust:defs.bzl", "rust_library")
+
+package(default_visibility = ["//rs:system-tests-pkg"])
+
+rust_library(
+    name = "utils",
+    testonly = True,
+    srcs = glob(["src/**/*.rs"]),
+    crate_name = "ic_consensus_threshold_sig_system_test_utils",
+    deps = [
+        # Keep sorted.
+        "//rs/canister_client",
+        "//rs/config",
+        "//rs/constants",
+        "//rs/nervous_system/common/test_keys",
+        "//rs/nns/cmc",
+        "//rs/nns/common",
+        "//rs/nns/constants",
+        "//rs/nns/governance",
```

Maybe consider being more explicit on [which rust_library](https://sourcegraph.com/github.com/dfinity/ic/-/blob/rs/nns/governance/BUILD.bazel) you're depending here:
```suggestion
        "//rs/nns/governance:governance",
```

---

Seems like `buildifier` doesn't like that 😄 
```
==================== Test output for //bazel:buildifier_test:
./rs/tests/consensus/tecdsa/utils/BUILD.bazel:
--- ./rs/tests/consensus/tecdsa/utils/BUILD.bazel	2024-07-25 12:36:32.159122491 +0000
+++ /tmp/buildifier-tmp-3096240069	2024-07-25 12:39:47.775372690 +0000
@@ -16,7 +16,7 @@
         "//rs/nns/cmc",
         "//rs/nns/common",
         "//rs/nns/constants",
-        "//rs/nns/governance:governance",
+        "//rs/nns/governance",
         "//rs/nns/test_utils",
         "//rs/prep",
         "//rs/registry/canister",
exit status 1
```